### DISCUSSION TITLE
ARM

### DISCUSSION THREAD

**Discussion by WINDROID-EMU** [ptitSeb/box86#882] [General]
Ask for DynaRec Block creation @0x601a20
Asked to Fill block 0xf1a0c360 with 0x601a20
protectDB 0x601a20 -> 0x601a20
isprotectedDB 0x601a20 -> 0x601a20 => 1
protectDB 0x601a20 -> 0x601a75

**Comment by KreitinnSoftware** [comment]
Send Entire Log

**Comment by KreitinnSoftware** [comment]
And gives more context for Sebatian understand this

**Comment by ptitSeb** [comment]
Ah yep. What this? Is there a question?
